src/main/java/org/generationcp/middleware/service/impl/study/StudyEntryServiceImpl.java

```diff
@@ -170,6 +175,11 @@ public void updateStudyEntryProperty(final int studyId, final StudyEntryProperty
 		}
 	}
 
+	@Override
+	public Boolean hasUnassignedStudyEntriesToPlot(final int studyId) {
```

Comment:
@mderamos I found this name hard to read, is it the same than asking if the study has observations generated?

Reply:
Hi @clarysabel, it's used for showing a warning when the study has a generated design and the user adds a new study entries. The added entries are not assigned to plots yet.

Reply:
@mderamos understood, in that case I think we should rename it to hasNotAssignedEntries or something simpler. If you still want to use Plot I suggest to use ObservationUnit instead but hasNotAssignedEntries is shorter and at the end there is no other object to assign entries besides the Observation Unit so it should be ok just make it shorter